libtenzir/builtins/contexts/geoip.cpp

```diff
@@ -531,9 +551,53 @@ struct v1_loader : public context_loader {
   }
 };
 
+struct v2_loader : public context_loader {
+  explicit v2_loader(record global_config)
+    : global_config_{std::move(global_config)} {
+  }
+
+  auto version() const -> int {
+    return 2;
+  }
+
+  auto load(chunk_ptr serialized) const
+    -> caf::expected<std::unique_ptr<context>> {
+    const auto* cache_dir
+      = get_if<std::string>(&global_config_, "tenzir.cache-directory");
+    const auto current_time = std::time(nullptr);
+    std::string temp_file_name = *cache_dir + std::to_string(current_time);
+    TENZIR_WARN(temp_file_name);
+    auto myfile
```

Comment:
```suggestion
    auto mmdb_file
```

The "my" prefix is often used in example, but we don't use it in production code. Recommendation: give it simply the most descriptive name you can think of, so that readers ideally know what it is by its name and type.

Reply:
Sorry, I opened for review on accident. Will do!